assets/l10n/en-US.ftl

```diff
@@ -990,6 +998,7 @@ label_no_updates_are_available_title = No updates are available
 label_no_users = No users
 label_no_welcome_message = Welcome message has not been added
 label_nobody = No one
+label_notes_divider = Features
```

Comment:
I'd suggest to call this label just that: `label_features`, as it might get used anywhere else like that.